portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,28 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> messages;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    //Hard code messages 
+    messages = new ArrayList<String>();
+    messages.add("How are you today?");
+    messages.add("I'm fine.");
+    messages.add("How are you?");
+
+    //Convert messages to json
+    Gson gson = new Gson();
```

Comment:
You can make this a `private static final` class member rather than creating a new instance on each `doGet`